### PR TITLE
delete data func

### PR DESCRIPTION
**Pull request by BarBenNaim**
we need to delete the Job DB before we write new data from the API's

### CODE REVIEW

File: ultimatejobweb/models.py [hunk 1]
```diff
@@ -13,5 +13,6 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
-
+    def delete_data():
```
Comment (simzacks) [inline ultimatejobweb/models.py]: This should be a classmethod.
The first parameter should be cls
the code would be 
cls.objects.all().delete()

**Review by simzacks — CHANGES_REQUESTED** [review]

File: ultimatejobweb/models.py (hunk 1 above)
Comment (Yarboa) [reply ultimatejobweb/models.py]: @BarBenNaim  please refer this sample PR
https://github.com/beyond-io/beyond-tutorial/pull/7/files#diff-d674dd14acda98e0ff388c113622f5224b22cde68f320bf7e6d39ce166214c09R11

Comment (BarBenNaim) [reply ultimatejobweb/models.py]: @simzacks What is the purpose of the class?

Comment (Yarboa) [reply ultimatejobweb/models.py]: In the way you are using here you should use self,
self means using instance methods.

While the use of @classmethod the 1st parameter is the class itself.
Here when working w/classes inherits from  model and are mapping of db data, we prefer to use  @classmethods working on the class itself

As sim already explained, 
@classmethod
def delete_data(cls):
    cls.objects.all().delete()

**Note:** the same for the push method

File: ultimatejobweb/models.py [hunk 2]
```diff
@@ -13,6 +13,7 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
-    def delete_data():
+    @classmethod
+    def delete_data(cls):
         Job.objects.all().delete()
```
Comment (Yarboa) [inline ultimatejobweb/models.py]: @BarBenNaim see the previous review comments:
you should call the method like that,
cls.objects.all().delete()

**Pull request merged**